packages/config-plugins/README.md

```diff
@@ -164,3 +164,148 @@ export const withIcons: ConfigPlugin = config => {
 Be careful using `withDangerousModifier` as it is subject to change in the future.
 The order with which it gets executed is not reliable either.
 Currently dangerous mods run first before all other modifiers, this is because we use dangerous mods internally for large file system refactoring like when the package name changes.
+
+## Static Plugins
+
+You don't need to migrate away from your existing `app.json` or `app.config.json` to use config plugins (although it is recommended). Plugins can be resolved automatically using the JSON `plugins` array:
+
+```js
+{
+  name: 'My App',
+  plugins: [
+    // A config plugin!
+    'expo-splash-screen'
+  ]
+}
+```
+
+Static plugins can be added in two different formats:
+
+```js
+{
+  plugins: [
+    // Long form with properties
+    {
+      resolve: 'my-plugin',
+      props: {
+        /* Passed as the second parameter to the config plugin */
+      },
+    },
```

Comment:
An alternative API that could be more inline with config plugins is:
```js
{
  plugins: [
    ['my-plugin', { /* props */}],
    './my-other-plugin',
  ]
}
```

This could be extended to work like `withPlugins`

```js

const withSomething = (config, props) => config;

{
  plugins: [
    [withSomething, {}],
    withSomething,
    'expo-splash-screen'
  ]
}
```